specs/experimental/zk-fault-dipute-game/overview.md

```diff
@@ -13,14 +13,88 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+[g-withdrawal]: ../../glossary.md#withdrawal
+
 ## Overview
 
+The ZK Fault Dispute Game (ZKFDG) is a new type of dispute game compatible with the OP Stack. In ZKFDG, the first
+disagreeing output root is found by the process called "dissection". Then, both derivation of L2 block from L1
```

Comment:
How about replacing `output root` with `root claim` in this document?

Reply:
Ok. I'll change it. But output root is also used in dispute game. They use both terms.